var/spack/repos/builtin/packages/shiro/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Shiro(MavenPackage):
+    """Apache Shiro is a powerful and easy-to-use Java security framework
+    that performs authentication, authorization, cryptography, and session
+    management. With Shiro’s easy-to-understand API, you can quickly and
```

Comment:
The smart apostrophe here is probably the source of the Python 2 unicode error:
```suggestion
    management. With Shiro's easy-to-understand API, you can quickly and
```